### ISSUE TITLE
Delay in triggering first on_step call when realtime=True

### ISSUE THREAD

**Issue by LostLucidity** [Dentosal/python-sc2#85]
Hello. As in the title, on_step does get trigger for me until the 11th second in game time. This does not happen when realtime=False. I've emptied out the on_step method with just a print statement to remove any potential computer lag with the same result.

**Comment by alexander-kazakov** [comment]
Yes, I have the same, but the interval is random. Sometimes a few seconds, sometimes even a whole minute...

**Comment by h3nnn4n** [comment]
Same here. It takes a few seconds before it runs if `realtime=True`

**Comment by LostLucidity** [comment]
This seems to be caused by VSCode. A couple of users on discord have seen this issue cleared when running on the console outside of VSCode.

**Comment by alexander-kazakov** [comment]
What is VSCode? I'm running via command line.

**Comment by LostLucidity** [comment]
Visual Studio Code is a popular IDE. You can run from their embedded command line. It seems a subset of this ticket's issue involves this IDE and using their terminal.

**Comment by pelletier2017** [comment]
I have the same issue and its from running the program in pycharm.
